runtime/src/iree/base/status.h

```diff
@@ -299,7 +299,7 @@ typedef struct iree_status_handle_t* iree_status_t;
 // We cut out all status storage code when not used.
 #if IREE_STATUS_FEATURES == 0
 #define IREE_STATUS_IMPL_MAKE_(code, ...) \
-  (iree_status_t)(uintptr_t)((code)&IREE_STATUS_CODE_MASK)
+  (iree_status_t)(uintptr_t)((code) & IREE_STATUS_CODE_MASK)
```

Comment:
clang-format on CI disagrees with this https://github.com/openxla/iree/actions/runs/8299606548/job/22715692098?pr=16791#step:5:16

Reply:
can update this
https://github.com/openxla/iree/blob/94a01083cdea284006c2187fc79156dd0862cb90/.github/workflows/lint.yml#L92-L107

Reply:
I'll give that a try

Reply:
What is your `clang-format --version`? CI is 12 and I have 16. Both want no spaces here.

Reply:
We can easily update to 13, 14, or 15.

Reply:
Would look something like this: https://github.com/openxla/iree/pull/16792